amqtt/session.py

```diff
@@ -159,16 +159,15 @@ def _init_states(self):
 
     @property
     def next_packet_id(self):
-        self._packet_id += 1
-        if self._packet_id > 65535:
-            self._packet_id = 1
+        self._packet_id = (self._packet_id % 65535) + 1
+        limit = self._packet_id
```

Comment:
Limit is added because previously, if the message just so happens to be 65535, then it would throw the "No free packet ID" error even though 1-65534 were still available